css/pseudoelements/demos/javascript-events/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>JavaScript-события для псевдоэлементов — Псевдоэлементы — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    html {
+      color-scheme: dark;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #18191C;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+    }
+
+    .quantity-change {
+      width: 160px;
+      height: 40px;
+      display: flex;
+      color: #333333;
+      background-color: #FFFFFF;
+      cursor: text;
+      margin: 0;
+      box-shadow: 0 3px 5px 0 rgb(0 0 0 / 0.1);
+      outline: 0;
+      padding: 0;
+      border: 1px solid #DADADA;
+      border-radius: 5px;
+      outline: none;
+      box-sizing: border-box;
+      align-items: center;
+      justify-content: space-between;
+      user-select: none;
+      }
+
+      .quantity-change::before, .quantity-change::after {
+        display: flex;
+        height: 40px;
+        width: 40px;
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        font-size: 24px;
+      }
+
+      .quantity-change::before {
+        content: '-';
```

Comment:
```suggestion
        content: '–';
```

Меняет минус на короткое тире, он по размеру больше похож на плюс.

Reply:
Привет. Я пока еще не все понимаю, мне так и не удалось отображать содержимое iframe в markdown.
Если отсутствие section ничего не сломает в отображении iframe, то можно и убрать.